docs/docs/persistent-volumes.md

```diff
@@ -69,6 +69,58 @@ The default `UpgradeStrategy` for a stateful application is a `minimumHealthCapa
 
 **Note:** For a stateful application, Marathon will never start more instances than specified in the `UpgradeStrategy`, and will kill old instances rather than create new ones during an upgrade or restart.
 
+## Creating a pod with local persistent volumes
+
+### Configuration options
+
+Let's configure a persistent volume with the following options:
+
+```json
+{
+  "name": "pst",
```

Comment:
Where should this go? I mean within a pod definition. Should this be under `volumes` in the root?

Reply:
Yes.

Reply:
@meln1k could you please make it more explicit?